lib/importer.js

```diff
@@ -12,8 +12,23 @@ var deleteDocuments = function(db, docIdsToDelete) {
   }
 };
 
-var storeSeq = function(db, seq) {
-  return db.query(format('UPDATE couchdb_progress SET seq = %L', seq));
+var getSeq = function(db, sourceId) {
+  return new rsvp.Promise(function(resolve, reject) {
```

Comment:
Why are you wrapping this in a manually contructed promise? Can you not just return the `db.one` and the `db.query` call below? Something like (not tested):

```js
var getSeq = function(db, sourceId) {
  var queryStmt = 'SELECT seq FROM couchdb_progress WHERE source_id = %L';
  return db.one(format(queryStmt, sourceId))
    .then(function(result) { return result.seq; })
    .catch(function() {
      var insertStmt = 'INSERT INTO couchdb_progress(seq, source_id) VALUES (%L, %L)';
      return db.query(format(insertStmt,  0, sourceId))
        .then(function() { return 0; });
    });
};
```